app.go

```diff
@@ -114,6 +118,7 @@ func NewApp() *App {
 		Action:       helpCommand.Action,
 		Compiled:     compileTime(),
 		Writer:       os.Stdout,
+		InterruptHandlerFunc: func(c *Context) {},
```

Comment:
The struct member should be called InterruptHandler, the type can be called func. Also I guess we could drop func from the type name too, but we should check what action does.


 I think the default should be something that cancels the context, and have an ExitOnInterrupt handler freely available in the library.